hack/display_cover_profile.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+: "${TMP_COVER_PROFILE:=/tmp/display_coverage.out}"
```

Comment:
what this does?

Reply:
sets default variable to that file, shorthand for `TMP_COVER_PROFILE${TMP_COVER_PROFILE:=/tmp/display_coverage.out}`